src/quoteservice/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM composer:2.4.1 AS build
+
+WORKDIR /tmp/
+COPY ./src/quoteservice/composer.json .
+
+RUN composer install \
+    --ignore-platform-reqs \
+    --no-interaction \
+    --no-plugins \
+    --no-scripts \
+    --prefer-dist
+
+FROM php:8.1-apache-buster
+
+# install GRPC (required for the OTel exporter)
+RUN apt-get -y update && apt install -y --no-install-recommends zlib1g-dev && \
+    pecl install grpc && \
```

Comment:
you should also install the protobuf extension, as the performance of the php-language implementation is pretty bad by comparison